cmd/controlplane/controller.go

```diff
@@ -44,12 +44,13 @@ func newControllerCommand() *cobra.Command {
 			version := versionpkg.GetVersion()
 			shardName := os.GetEnv("SHARD_NAME", "")
 
-			startupLogEntry := log.WithFields(log.Fields{
-				"version": version.Version,
-				"commit":  version.GitCommit,
-			})
+			log := logging.LoggerFromContext(ctx)
+			startupLogEntry := log.WithValues(
+				"version", version.Version,
+				"commit", version.GitCommit,
+			)
 			if shardName != "" {
-				startupLogEntry = startupLogEntry.WithField("shard", shardName)
+				startupLogEntry = startupLogEntry.WithValues("shard", shardName)
 			}
 			startupLogEntry.Info("Starting Kargo Controller")
```

Comment:
I think all of this can be written more cleanly with logr than with logrus. Something like this?

```
keyVals := []any{"version", version.Version, "commit", version.GitCommit}
if shardName != "" {
	keyVals = append(keyVals, "shard", shardName)
}
log.Info("Starting Kargo Controller", keyVals...)
```

Reply:
Thanks, modified in [f9266aa](https://github.com/akuity/kargo/pull/1791/commits/f9266aa3804467d1eb443382fadfdf27be5064ec)